data/keymaps/i386/qwertz/de_alt_UTF-8.map

```diff
@@ -160,7 +163,7 @@ shift alt keycode   2 = U+00AC  # notsign
 shift alt keycode   3 = U+201D  # right double quote
 shift alt keycode   4 = numbersign
 shift alt keycode   5 = sterling
-shift alt keycode   6 = U+FB01  # fi ligature
+#K#shift alt keycode   6 = U+FB01  # fi ligature
```

Comment:
What is the point of commenting out these lines and not deleting them completely?

Since this bug was in keymap initially starting in 2009. I'm sure no one used it. I think it's better to remove it completely.

Reply:
These codes are correct, but currently not supported by the kernel. In future, things may change.

I am not against a complete dropping of the keymap. It is one of the "alternative" keyboards. This time it is a character-rich German keyboard.

It probably has no more than 1 user.